README.md

```diff
@@ -4,6 +4,65 @@ This research repository maintains the code and the results for the research pap
 
 _"Set theory has become the standard foundation for mathematics, as every mathematical object can be viewed as a set."_ -[Stanford Encyclopedia of Philosophy](https://plato.stanford.edu/entries/set-theory/)
 
+# Install Package from PyPI and Use
+To install the package, please run:
+```pip install setlexsem```
+You can generate the dataset by:
+```python
+from setlexsem.generate.generate_data import generate_data
+generate_data(
+    set_types=["numbers"],
+    n=[10],
+    m=[1,2],
+    item_len=[1],
+    decile_group=None,
+    swap_status=None,
+    overlap_fraction=[None],
+    seed_value=292,
+    number_of_data_points= 3
+)
+# --- or, this can be config based:
+config = {
+    "set_types": ["numbers"],
+    "n": [10],
+    "m": [10],
+    "item_len": [2],
+    "decile_group": None,
+    "swap_status": None,
+    "overlap_fraction": [0.5],
+}
+out_data = generate_data(
+    config=config,
+    number_of_data_points= 5,
+    seed_value=292
+)
+```
+You can generate the prompts by:
+```python
+from setlexsem.generate.generate_prompts import generate_prompts
+
+generate_prompts(
```

Comment:
Show an example of saving the prompts to disk and explain how the prompts can be used.

Reply:
will come back to this after the changes are finalized